src/main/target/AIR32/target.h

```diff
@@ -112,3 +112,5 @@
 #define USABLE_TIMER_CHANNEL_COUNT 10
 #define USED_TIMERS             (TIM_N(1) | TIM_N(2) | TIM_N(3) | TIM_N(15) | TIM_N(17))
 
+
+#define RUNCAM_SPLIT_SUPPORT
```

Comment:
`RUNCAM_SPLIT_SUPPORT` should be renamed `USE_RUNCAM_SPLIT_SUPPORT` and defined in `common_fc_pre.h`, not every `target.h` file.